app/components/global/layout/media.js

```diff
@@ -0,0 +1,20 @@
+import { css } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
+
+const sizes = {
+  mobile: theme.breakpoints[0],
+  tabletPortrait: theme.breakpoints[1],
+  tabletLandscape: theme.breakpoints[2],
+  desktop: theme.breakpoints[3],
+}
+
+const media = Object.keys(sizes).reduce((acc, label) => {
+  acc[label] = (...args) => css`
+    @media (min-width: ${sizes[label]}px) {
+      ${css(...args)};
+    }
+  `
+  return acc
+}, {})
+
+export default media
```

Comment:
It's a bit unexpected that `media.mobile<style>` actually applies that style to everything *but* mobile.